python/README.md

```diff
@@ -0,0 +1,36 @@
+# truelayer-signing
+Python package to produce & verify TrueLayer API requests signatures.
+
+```
+tl_signature = sign_with_pem(KID, PRIVATE_KEY) \
+    .set_method(HttpMethod.POST) \
+    .set_path(path) \
+    .add_header("Idempotency-Key", idempotency_key) \
+    .set_body(body) \
+    .sign()
```

Comment:
One thing I was wondering about python is if this builder is idiomatic? My experience with python you tend to have a single method call with multiple (maybe optional) arguments. Wdyt?

For similar reasons I used a different API style for nodejs: https://github.com/TrueLayer/truelayer-signing/tree/main/nodejs#usage

Reply:
I think taking a big list of parameters is more common in python due to named arguments but I wouldn't say the builder is not idiomatic, python code varies too much for there to be any standards in my opinion. I do lean slightly in favour of a single method

Reply:
Ok as long as we have some angle to call it idiomatic then I'm ok with the builder style.